src/ArrowTypes/src/ArrowTypes.jl

```diff
@@ -344,6 +344,9 @@ function ToArrow(x::A) where {A}
         for i = 2:length(x)
             @inbounds T = promoteunion(T, typeof(toarrow(x[i])))
         end
+        if T === Missing
```

Comment:
So there's an interesting case to consider here: `Any[missing, missing]`. The array eltype is `Any`, so we hit this codepath, and `T` is `Missing`, so we'll hit this new branch, so uh-oh, that might do something bad, right? Well, `ArrowTypes.default(Any)` returns `nothing`, and `toarrow(nothing)` returns.......`missing`! So we end up with the right `T` in the end, albeit somewhat unexpectedly? I would only worry a bit if we end up changing `toarrow(::Nothing) = missing`.

For now, can we just add a test for this case and then deal w/ it later if necessary?

Reply:
Sounds good. I'll try to take care of this and the other outstanding things this week.